### DISCUSSION TITLE
Automatic answering of calls

### DISCUSSION THREAD

**Discussion by msxmine** [element-hq/element-meta#679] [Ideas]
FEATURE REQUEST:
It would be great if riot added some kind of auto answer option. (When a call to a client with auto-answer configured is performed, the call is automatically accepted). 

Preferably the user could configure some kind of whitelist of callers, for which this functionality would apply  

This feature is usually found in more traditional voip apps (Skype, Ring.cx and Wire all implement it), and it also enables some new use-cases

For example:

- In-browser simple video surveillance
- Remote control of embedded IOT devices
- Better accesibility for people not able to answer the call themselves

**Comment by lampholder** [comment]
We used this feature a lot on Skype for team members dialling into a shared terminal from remote locations - I'd like to see this :)

**Comment by Artiom-M** [comment]
Yes, definitely a must have option. This shouldn't be too much work to add it.

**Comment by Boteium** [comment]
the first thing I googled after install this app
would like to see this being implemented

**Comment by kittykat** [comment]
Moving this issue to discussions in Element meta as we need to make a cross platform decision on how to proceed :+1: 

I believe video rooms now covers the "drop in" use case

**Reply by karlabbott** [reply]
Yes, but for people that want to bridge SIP into Element, we do still need a way to auto-answer incoming SIP calls -- especially if it's from somebody we've never heard from before as each new SIP phone number is a new room on the Matrix side.
